cmd/cmd/serviceInfo.go

```diff
@@ -119,33 +119,33 @@ func executeGetServiceCmd(servicename string) {
 // @return error
 func GetServiceInfo(name string) (*utils.Service, error) {
 
-    finalUrl := utils.RESTAPIBase + utils.PrefixServices + "/" + name
+	finalUrl := utils.RESTAPIBase + utils.PrefixServices + "/" + name
 
-    utils.Logln(utils.LogPrefixInfo+"URL:", finalUrl)
+	utils.Logln(utils.LogPrefixInfo+"URL:", finalUrl)
 
-    headers := make(map[string]string)
-    // headers[utils.HeaderAuthorization] = utils.HeaderValueAuthPrefixBearer + " " + accessToken
+	headers := make(map[string]string)
+	// headers[utils.HeaderAuthorization] = utils.HeaderValueAuthPrefixBearer + " " + accessToken
 
-    resp, err := utils.InvokeGETRequest(finalUrl, headers)
+	resp, err := utils.InvokeGETRequest(finalUrl, headers)
 
-    if err != nil {
-        utils.HandleErrorAndExit("Unable to connect to "+finalUrl, err)
-    }
+	if err != nil {
+		utils.HandleErrorAndExit("Unable to connect to "+finalUrl, err)
+	}
 
-    utils.Logln(utils.LogPrefixInfo+"Response:", resp.Status())
+	utils.Logln(utils.LogPrefixInfo+"Response:", resp.Status())
 
-    if resp.StatusCode() == http.StatusOK {
-        serviceResponse := &utils.Service{}
-        unmarshalError := xml.Unmarshal([]byte(resp.Body()), &serviceResponse)
+	if resp.StatusCode() == http.StatusOK {
+		serviceResponse := &utils.Service{}
+		unmarshalError := xml.Unmarshal([]byte(resp.Body()), &serviceResponse)
 
-        if unmarshalError != nil {
-            utils.HandleErrorAndExit(utils.LogPrefixError+"invalid XML response", unmarshalError)
-        }
+		if unmarshalError != nil {
+			utils.HandleErrorAndExit(utils.LogPrefixError+"invalid XML response", unmarshalError)
+		}
 
-        return serviceResponse, nil
-    } else {
-        return nil, errors.New(resp.Status())
-    }
+		return serviceResponse, nil
+	} else {
+		return nil, errors.New(resp.Status())
+	}
```

Comment:
Lot of code in these methods are duplicated. Shall we refactor it to avoid code duplication?